webpack/modules/raml-validator-loader/lib/raml-validator-loader.js

```diff
@@ -80,7 +80,7 @@ module.exports =
 	  // Override the default filesystem resolver in order to:
 	  //
 	  // 1) Provide a custom content for the source file
-	  // 2) Track dependent files in order for webpack to invalidate caches
+	  // 2) Track dependant files in order for webpack to invalidate caches
```

Comment:
Alright, I'll cave here. It's not important, as long as we make sure to catch all of them when we put up the next PR. I would have been nice to have this PR cleaned up, though, which would have saved us time, even though it might have taken you a little extra time to cherry-pick that commit.